.github/workflows/check-nix-format.yml

```diff
@@ -23,8 +23,11 @@ jobs:
         with:
           # explicitly enable sandbox
           extra_nix_config: sandbox = true
+          # fix a commit from nixpkgs-unstable to avoid e.g. building nixfmt
+          # from staging
+          nix_path: nixpkgs=https://github.com/NixOS/nixpkgs/archive/4b455dc2048f73a79eb3713f342369ff58f93e0b.tar.gz
```

Comment:
Why an arbitrary commit and not a moving target like nixos-unstable?

Reply:
We can't do that because it means that CI will suddenly break when the channel updates with a new version of nixfmt, with any sort of formatting changes.